src/com/spikes2212/utils/PIDSettings.java

```diff
@@ -41,127 +44,121 @@ public PIDSettings(double KP, double KI, double KD, double tolerance, double wai
+	 * tolerance for the amount of time specified by {@link #setWaitTime(double)}.
 	 *
 	 * @param tolerance
-	 *            The new tolerance to set. If 0 and the WaitTime is not 0, this
-	 *            PID loop will never end unless you cancel it.
-	 * @see PIDController#setAbsoluteTolerance(double)
+	 *            the new tolerance to set. If tolerance is 0 and the WaitTime is
+	 *            not 0, this PID loop will never end, unless you cancel it.
 	 */
 	public void setTolerance(double tolerance) {
 		this.tolerance = tolerance;
 	}
 
 	/**
-	 * Gets the Proportional coefficient of the PID loop using this settings.
+	 * Gets the Proportional coefficient of the PID loop using these
+	 * {@link PIDSettings}.
 	 *
-	 * @return the current Proportional coefficient.
-	 * @see PIDController#getP()
+	 * @return The current Proportional coefficient.
 	 */
 	public double getKP() {
 		return KP;
 	}
 
 	/**
-	 * Sets the Proportional coefficient of the PID loop using this settings.
+	 * Sets the Proportional coefficient of the PID loop using these
+	 * {@link PIDSettings}.
 	 *
 	 * @param KP
 	 *            the new Proportional coefficient.
-	 * @see PIDController#setPID(double, double, double)
 	 */
 	public void setKP(double KP) {
 		this.KP = KP;
 	}
 
 	/**
-	 * Gets the Integral coefficient of the PID loop using this settings.
+	 * Gets the Integral coefficient of the PID loop using these
+	 * {@link PIDSettings}.
 	 *
-	 * @return the current Integral coefficient.
-	 * @see PIDController#getI()
+	 * @return The current Integral coefficient.
 	 */
 	public double getKI() {
 		return KI;
 	}
 
 	/**
-	 * Sets the Integral coefficients of the PID loop using this settings.
+	 * Sets the Intergral coefficient of the PID loop using these
+	 * {@link PIDSettings}.
 	 *
-	 * @param KI
+	 * @param KP
 	 *            the new Integral coefficient.
-	 * @see PIDController#setPID(double, double, double)
 	 */
 	public void setKI(double KI) {
 		this.KI = KI;
 	}
 
 	/**
-	 * Gets the Differential coefficient of the PID loop using this settings.
+	 * Gets the Differential coefficient of the PID loop using these
+	 * {@link PIDSettings}.
 	 *
-	 * @return the current Differential coefficient.
-	 * @see PIDController#getD()
+	 * @return The current Differential coefficient.
 	 */
 	public double getKD() {
 		return KD;
 	}
 
 	/**
-	 * 
-	 * Sets the Differential coefficient of the PID loop using this settings.
+	 * Sets the Differential coefficient of the PID loop using these
+	 * {@link PIDSettings}.
 	 *
-	 * @param KD
+	 * @param KP
 	 *            the new Differential coefficient.
-	 * @see PIDController#setPID(double, double, double)
 	 */
 	public void setKD(double KD) {
 		this.KD = KD;
 	}
 
 	/**
-	 * Gets the time this PID loop will wait while within tolerance of the
-	 * setpoint before ending.
-	 * <p>
-	 * The PID control of the subsystem continues while waiting
-	 * </p>

```

Comment:
why did you remove this line? (The PID control of the subsystem continues while waiting)